resources/options_lookup.tsv

```diff
@@ -1686,12 +1689,23 @@ Misc Hot Tub Spa	None	ResidentialMiscLargeUncommonLoads	has_hot_tub_heater_elec=
 Misc Hot Tub Spa	National Average	ResidentialMiscLargeUncommonLoads	has_hot_tub_heater_elec=true	has_hot_tub_heater_gas=true	has_hot_tub_pump=true	hot_tub_heater_elec_annual_energy=1027.3	hot_tub_heater_elec_mult=0.048	hot_tub_heater_gas_annual_energy=81	hot_tub_heater_gas_mult=0.011	hot_tub_pump_annual_energy=1014.1	hot_tub_pump_mult=0.059	hot_tub_scale_energy=true	"hot_tub_weekday_sch=0.024, 0.029, 0.024, 0.029, 0.047, 0.067, 0.057, 0.024, 0.024, 0.019, 0.015, 0.014, 0.014, 0.014, 0.024, 0.058, 0.126, 0.122, 0.068, 0.061, 0.051, 0.043, 0.024, 0.024"	"hot_tub_weekend_sch=0.024, 0.029, 0.024, 0.029, 0.047, 0.067, 0.057, 0.024, 0.024, 0.019, 0.015, 0.014, 0.014, 0.014, 0.024, 0.058, 0.126, 0.122, 0.068, 0.061, 0.051, 0.043, 0.024, 0.024"	"hot_tub_monthly_sch=0.921, 0.928, 0.921, 0.915, 0.921, 1.160, 1.158, 1.158, 1.160, 0.921, 0.915, 0.921"																																			
 Misc Hot Tub Spa	Electric	ResidentialMiscLargeUncommonLoads	has_hot_tub_heater_elec=true	has_hot_tub_heater_gas=false	has_hot_tub_pump=true	hot_tub_heater_elec_annual_energy=1027.3	hot_tub_heater_elec_mult=1.0	hot_tub_heater_gas_annual_energy=81	hot_tub_heater_gas_mult=0.011	hot_tub_pump_annual_energy=1014.1	hot_tub_pump_mult=1.0	hot_tub_scale_energy=true	"hot_tub_weekday_sch=0.024, 0.029, 0.024, 0.029, 0.047, 0.067, 0.057, 0.024, 0.024, 0.019, 0.015, 0.014, 0.014, 0.014, 0.024, 0.058, 0.126, 0.122, 0.068, 0.061, 0.051, 0.043, 0.024, 0.024"	"hot_tub_weekend_sch=0.024, 0.029, 0.024, 0.029, 0.047, 0.067, 0.057, 0.024, 0.024, 0.019, 0.015, 0.014, 0.014, 0.014, 0.024, 0.058, 0.126, 0.122, 0.068, 0.061, 0.051, 0.043, 0.024, 0.024"	"hot_tub_monthly_sch=0.921, 0.928, 0.921, 0.915, 0.921, 1.160, 1.158, 1.158, 1.160, 0.921, 0.915, 0.921"																																			
 Misc Hot Tub Spa	Gas	ResidentialMiscLargeUncommonLoads	has_hot_tub_heater_elec=false	has_hot_tub_heater_gas=true	has_hot_tub_pump=true	hot_tub_heater_elec_annual_energy=1027.3	hot_tub_heater_elec_mult=1.0	hot_tub_heater_gas_annual_energy=81	hot_tub_heater_gas_mult=1.0	hot_tub_pump_annual_energy=1014.1	hot_tub_pump_mult=1.0	hot_tub_scale_energy=true	"hot_tub_weekday_sch=0.024, 0.029, 0.024, 0.029, 0.047, 0.067, 0.057, 0.024, 0.024, 0.019, 0.015, 0.014, 0.014, 0.014, 0.024, 0.058, 0.126, 0.122, 0.068, 0.061, 0.051, 0.043, 0.024, 0.024"	"hot_tub_weekend_sch=0.024, 0.029, 0.024, 0.029, 0.047, 0.067, 0.057, 0.024, 0.024, 0.019, 0.015, 0.014, 0.014, 0.014, 0.024, 0.058, 0.126, 0.122, 0.068, 0.061, 0.051, 0.043, 0.024, 0.024"	"hot_tub_monthly_sch=0.921, 0.928, 0.921, 0.915, 0.921, 1.160, 1.158, 1.158, 1.160, 0.921, 0.915, 0.921"																																			
+Misc Hot Tub Spa Pump																																																		

```

Comment:
Are these two lines supposed to be here?

Reply:
Gah, nope. Good catch.